### DISCUSSION TITLE
w sublayer will not trigger

### DISCUSSION THREAD

**Discussion by m-collins** [mxstbr/karabiner#33] [Q&A]
Is the `w` sublayer + `semicolon` functional for others?
Initially, I thought I had made a configuration error when customising my config. However, when I run master, it doesn't  trigger. No keys are recorded when viewing karabiner event viewer.

What's strange, when using semicolon on the `s` sublayer, everything works as expected.

**Comment by m-collins** [comment]
Note, I can very occasionally get the action to be triggered. If I spam the keyboard with this combo, it occasionally is recorded in karabiner event viewer and triggered.

**Comment by mxstbr** [comment]
Hm, odd, it works perfectly for me. Do you have anything else managing your keyboard?
